fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/TriState.java

```diff
@@ -136,4 +136,21 @@ public <X extends Throwable> boolean orElseThrow(Supplier<X> exceptionSupplier)
 
 		throw exceptionSupplier.get();
 	}
+
+	/**
+	 * {@return a parsed TriState from a system property}
+	 *
+	 * @param property the system property
+	 */
+	public static TriState fromProperty(String property) {
```

Comment:
Maybe from**System**Property? `fromProperty` on an enum sounds more like parsing a block/fluid state property to me.

Reply:
Done.